moduleframework/mtf_environment.py

```diff
@@ -40,8 +41,9 @@
     env = EnvNspawn()
 
 def mtfenvset():
+    print_info("Preparing environment ...")
```

Comment:
Would it be possible to add also information from env.prepare_env() function? May be more details.

Reply:
there is 'env.prepare_env' duplicated in backends for : EnvDocker/EnvRpm/EnvNspawn.
It throws some user info during process.

Also I changed package installation little bit, where it shows some messages what is installed and shows nice error when installation failed